manifest.json

```diff
@@ -35,7 +36,7 @@
         "2.0.1",
         "2.0.2",
         "3.0.0",
-	"3.0.1"
+	      "3.0.1"
```

Comment:
Very nitpicky (I know) but there's a couple of extra tabs here... Not going to block merging because of pipelines but we should address it on the next PR for sure

Reply:
Not sure how those ended up there, I just decided to remove them now.